internal/cli/config/edit.go

```diff
@@ -47,12 +47,11 @@ func EditBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "edit",
 		Short: "Opens the the config with the default text editor.",
-		Long:  `Will use the default editor to open the config file. You can use EDITOR or VISUAL envs to change the default.`,
+		Long:  `Uses the default editor to open the tool config file. You can use EDITOR or VISUAL envs to change the default.`,
```

Comment:
Question: why did we add "tool" here?

Reply:
why not? just to make it clearer but open to suggestions, this commands was undocumented so it has never been reviewed